l10n_br_fiscal/models/document.py

```diff
@@ -680,10 +680,22 @@ def document_comment(self):
                 record._document_comment_vals())
             record.line_ids.document_comment()
 
+    def _get_email_template(self, state):
+        return self.document_type_id.document_email_ids.filtered(
+            lambda d: d.state_edoc == state
+            and self.issuer == d.issuer).mapped('email_template_id')
+
+    def send_email(self, state):
+        email_template = self._get_email_template(state)
+        if email_template:
+            email_template.send_mail(self.id)
```

Comment:
Na v12 quando vc nao bota decorador num metodo, por padrao ele eh um @multi. Nisso me parece que os metodos _get_email_template e send_email nao iriam funcionar com uma lista de registros (respectivamente por causa dos `self.document_type_id` e `self.id`). Eu acho que faltou um for `document in self:` ou um `self.ensure_one()`

Reply:
@rvalyi aqui eh melhor colocar um self.ensure_one() tbm ?

Reply:
Eu diria que tanto faz ja que tem um no _get_email_template e no send_mail depois. Ou seja ta legal assim para mim. Eu so nao aprovei ainda que fiquei de revisar o dominio de busca que vc colcou me parece estranho mas talvez sou eu que nao pensei direito ainda.

Reply:
Heheheh por favor verifique, eu sinceramente fiquei em dúvida sobre o domínio.

Reply:
caramba odeio esses dominios compostos assim rs. Mas ta certo. Uso esse post qdo eu esqueco alias https://www.surekhatech.com/blog/working-with-domain-and-polish-notations-in-od-1